sdk/src/main/java/com/sygic/travel/sdk/Parser.kt

```diff
@@ -1,15 +0,0 @@
-package com.sygic.travel.sdk
-
-import com.google.gson.FieldNamingPolicy
-import com.google.gson.GsonBuilder
-import java.lang.reflect.Type
-
-object Parser {
```

Comment:
Should be probably removed in other commit

Reply:
:heavy_check_mark: